app/src/main/java/com/mapbox/services/android/navigation/testapp/example/ui/ExamplePresenter.kt

```diff
@@ -192,6 +196,11 @@ class ExamplePresenter(private val view: ExampleView, private val viewModel: Exa
     viewModel.route.observe(owner, Observer { onRouteFound(it) })
     viewModel.progress.observe(owner, Observer { onProgressUpdate(it) })
     viewModel.milestone.observe(owner, Observer { onMilestoneUpdate(it) })
+    viewModel.geocode.observe(owner, Observer {
+      it?.features()?.first().let {
+        onDestinationFound(it!!)
```

Comment:
I don't think we need the `!!` in `it!!` here - by updating to `first()?.let`, this should only execute if not null